src/pages/Transactions/index.tsx

```diff
@@ -85,27 +86,27 @@ const _TransactionsPage = ({
 
   const createButtonText = (): string => {
     if (totalTxCount === 1) {
-      return 'Initiate the transaction.';
+      return 'Confirm deposit';
```

Comment:
These strings should be wrapped in `intl.formatMessage()`

Reply:
Will push a fix